layouts/default.vue

```diff
@@ -3,8 +3,28 @@
     <ModalJoinDedi />
     <ModalCriteriaMitra />
     <ModalCompetitionFlowDetail />
-    <Header />
+    <SidebarMobile :show="sidebarOpen" @close-sidebar="clsoeSidebar" />
```

Comment:
Hello @doohanas I think there is a typo on the method name, clsoeSidebar -> closeSidebar